playbooks/bootstrap-ssh.yml

```diff
@@ -2,31 +2,27 @@
 - hosts: 127.0.0.1
   connection: local
   gather_facts: false
-  vars:
-    ansible_become: false
   tasks:
     - name: Create SSH directory if it doesn't exist
       file:
         path: "{{ lookup('env','HOME') + '/.ssh' }}"
         state: directory
-    # Requires Ansible v2.8
-#    - name: Generate default SSH keypair if it doesn't exist
-#      openssh_keypair:
-#        path: "{{ lookup('env','HOME') + '/.ssh/id_rsa' }}"
+
     - name: Generate default SSH keypair if it doesn't exist
-      command: ssh-keygen -q -t rsa -f "{{ lookup('env','HOME') + '/.ssh/id_rsa' }}" -C "" -N ""
+      command: ssh-keygen -q -t rsa -f "{{ lookup('env','HOME') + '/.ssh/id_rsa_ansible' }}" -C "" -N ""
```

Comment:
I don't really like this, now I have to specify the SSH key every time I access the server

Reply:
Ok, I can set it back to `id_rsa`. I don't have one of those on my system, and my SSH config isn't configured to use it. So, for me, I was planning to add a line to my inventory file if I ever needed to use this playbook. Seemed like I might as well be explicit about the purpose of the key.

But you're probably right that many people have `IdentityFile ~/.ssh/id_rsa` in their ssh configs. Which would make this playbook "just work".

Reply:
The default SSH client on any system I've ever used has looked for that file by default, which is why I used it here.

I like this in theory but it just adds extra steps most of the time I think. Maybe we could make it a var and set `id_rsa` as the default?